[DRAFT] OpenSWATH for Metabolomics

oliveralka opened the pull request:
Draft on how to use the DIAMetAlyzer workflow. 

The current workflow depicts the whole analysis of the benchmarking data - this is probably too much for a short tutorial session. 

It will help to understand the workflow, reanalyse the data and probably use it with own data. 

Let me know what you think!

oliveralka commented:
Link to PDF version (20200514):
https://drive.google.com/open?id=1Snar3Vhpto02TisAV08OB0CmAIDdfcKW

oliveralka commented:
[handout_add_DIAMetAlyzer_20200723.pdf](https://github.com/OpenMS/Tutorials/files/4964401/handout_add_DIAMetAlyzer_20200723.pdf)
@jpfeuffer Please let me know what you think when you have time. If everything is alright we can merge.

jpfeuffer commented:
So, 7.2. you could make some paragraphs.
In the workflow, you could merge all the Input variables into a Wrapped Metanode (or whatever this is called these days haha). Then you can make a little Wizard for the user so they can enter those parameters with a little help/description.

oliveralka commented:
Ok, that is pretty neat idea - I will give it a try!

oliveralka commented:
Ok, I added the paragraphs. For the configuration, I switched to a component with configuration nodes, which will open a configuration menu directly. 

I tested the interactive view using widgets, but I found that if everything else is "normally" configurated (e.g. input files) the opening the browser to configure four additional variables is not very intuitive.

jpfeuffer commented:
Yes, for input files you would another input variable and pass it and for multiple files this also not straightforward, so I usually don't use it for files as well.
It was a suggestion because it was four variable nodes and one might save work. Yes, it is different from the usual configuration procedure. Complete KNIME newbies might struggle. We have a small chapter on meta nodes I think.
Concerning paragraphs: looks good. I did not expect section numbers, just paragraphs, but this is fine too of course.

oliveralka commented:
@jpfeuffer Thanks for taking a look - switched to paragraphs - looks better! Should be ready to merge now!

jpfeuffer commented on Handout/openswathmetabo.tex:
```diff
@@ -0,0 +1,213 @@
+\textbf{parameter} & \textbf{explanation} \\ \hline
+\textit{mass\_error\_value} & Tolerance allowed for accurate mass search. \\
+\textit{ionization\_mode} &  Positive or negative ionization mode. \\
+\textit{mass\_error\_value} & Tolerance allowed for accurate mass search. \\
```
duplication

jpfeuffer commented on Handout/openswathmetabo.tex:
```diff
@@ -0,0 +1,213 @@
+\end{tabular*}
+\end{center} 
+
+\noindent If you are analysing a lot of big DIA mzML files ~3-20GB per File, it makes sense to  change how OpenSWATH processes the spectra. 
```
\approx

jpfeuffer replied:
double space "to  change"

jpfeuffer commented on Handout/openswathmetabo.tex:
```diff
@@ -0,0 +1,213 @@
+\end{figure}
+
+\subsection{Run the Workflow}
+These steps need to be followed to run the workflow successfully
```
":"?

jpfeuffer commented on Handout/openswathmetabo.tex:
```diff
@@ -0,0 +1,213 @@
+\url{https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Tutorials/Data/DIAMetAlyzer/}
+
+\subsection{Example Workflow}
+Example workflow for the usage of the DIAMetAlyzer Pipeline in KNIME (see Fig. \ref{fig:oswm_example_wf}). Inputs are the SWATH-MS data in profile mode (.mzML), a path for saving the new target-decoy assay library, the SIRIUS 4.0.1 executable, the DDA data (.mzML), custom libraries and adducts for AccurateMassSearch, the min/max fragment mass-to-charge to be able to restrict the mass of the trans ...
```
Missing punctuation at end?

jpfeuffer commented on Handout/openswathmetabo.tex:
```diff
@@ -0,0 +1,213 @@
+pyprophet merge  --template path_to_target-decoy_assay_library.pqp --out merged.osw  ./*.osw
+\end{lstlisting}
+
+\noindent Afterwards, the results are scored using the MS1 and MS2 leves and filter for metabolomics scores, which have a low correlation. 
```
levels
